tests/baselines/reference/privateNameFieldCallExpression.js

```diff
@@ -35,15 +36,17 @@ class A {
     }
     test() {
         var _a;
+        var _b;
         __classPrivateFieldGet(this, _fieldFunc).call(this);
+        (_a = __classPrivateFieldGet(this, _fieldFunc)) === null || _a === void 0 ? void 0 : _a(this);

```

Comment:
I don't think this fixes the issue. The expectation is that `this.#fieldFunc` will be called with the correct `this`, but instead its called with `this` as an argument. In other words, it should read like this instead:

```js
(_a = __classPrivateFieldGet(this, _fieldFunc)) === null || _a === void 0 ? void 0 : _a.call(this);
```